### PR TITLE
Modified path of metadata tool in ConfigPath Class to match with path in Configuration.properties

### PR DESCRIPTION
**Pull request by Laura4lilavati**

### CODE REVIEW

**Review by MirkoFer-622 — APPROVED** [review]

**Review by samuelloza — APPROVED** [review]

**Review by eli7890 — APPROVED** [review]

**Review by julia-jpg — APPROVED** [review]

File: src/main/java/org/fundacionjala/converter/model/command/VideoModel.java [hunk 1]
```diff
@@ -181,7 +172,12 @@ private String getNameFile(final String path, final VideoParameter videoParamete
         list = new ArrayList<>();
         list.add(convert(videoParameter));
         if (videoParameter.isExtractThumbnail()) {
-            list.add(extractThumbnail(videoParameter));
+            String[] parts = videoParameter.getFormat().split(DOT_SEPARATOR);
+            if (!parts[1].equals(GIF)) {
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/command/VideoModel.java]: try to use constants

File: src/main/java/org/fundacionjala/converter/model/command/VideoModel.java [hunk 2]
```diff
@@ -97,21 +116,15 @@
...
-        String detail = "v";
-        int name = 0;
         for (String path : outputFiles) {
-            checksum = checksumMD5.getMD5(path);
-            listMetadataParameters.add(new MetadataParameter(inputFileMetadata + getNameFile(path, videoParameter), format, detail, outputFile + name + "", checksum));
-            name++;
+            File file = new File(path);
+            String checksum = new ChecksumMD5().getMD5(path);
+            String[] parts = file.getName().split(DOT_SEPARATOR);
+            String outputFile = parts[0] + parts[1] + DASH + META;
```
Comment (roht0515) [inline src/main/java/org/fundacionjala/converter/model/command/VideoModel.java]: use constants please

File: src/main/java/org/fundacionjala/converter/model/command/VideoModel.java (hunk 1 above)
Comment (Laura4lilavati) [reply src/main/java/org/fundacionjala/converter/model/command/VideoModel.java]: it's done

File: src/main/java/org/fundacionjala/converter/model/command/VideoModel.java (hunk 2 above)
Comment (Laura4lilavati) [reply src/main/java/org/fundacionjala/converter/model/command/VideoModel.java]: it's done

**Review by roht0515 — APPROVED** [review]

**Pull request merged**